src/js/BookReader/options.js

```diff
@@ -189,17 +189,10 @@ export const DEFAULT_OPTIONS = {
   getEmbedCode: null,
 
   controls: {
-    onePage: {
+    viewmode: {
```

Comment:
Ahhh, this is a breaking change 😓 Is there a way we could use the old syntax here? And then in the code e.g. `if (controls?.onePage?.visible) { ... }`

Reply:
I was trying to retain the option structure, but the issue I ran into was that I'd have to add properties to the existing control objects to know which controls to remove from the array when the viewmode button is created. Since this control object allows you to override every property, there was no way to map the particular control to a view mode.

This option was added specifically for the audio player's liner notes BR instance, and when Petabox is upgraded to use this version, the options will change to achieve the same result. Beyond that, I was developing with Brenton's comment of ultimately developing this project for our own use in the back of my mind. If we need to make breaking changes so we can develop the features we want, then that's what we do.

On the other hand, to make the old options work and the new viewmode switcher work, it could be an either/or option. If `viewmode` is one of the controls, this ignores the other individual view mode control options. That will take a little work and some conditional logic in the Navbar template, but it could be done.

Reply:
:+1: The current solution lgtm!